app/code/Magento/Eav/Model/ResourceModel/Entity/Attribute/Group/Collection.php

```diff
@@ -30,12 +30,13 @@ protected function _construct()
     /**
      * Set Attribute Set Filter
      *
-     * @param int $setId
+     * @param string $setId
+     * @param string $conditionType
      * @return $this
      */
-    public function setAttributeSetFilter($setId)
+    public function setAttributeSetFilter($setId, $conditionType = 'eq')
```

Comment:
Unfortunately, we are not allowed to change public methods signature according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)

Reply:
@sidolov thakns for your comment. Can i suggest the next solution:
`app/code/Magento/Eav/Model/ResourceModel/Entity/Attribute/Group/Collection.php`
```
    /**
     * Set Attribute Set Filter
     *
     * @deprecated since do not sets ConditionType
     * @see setAttributeSetTypeSetFilter($setId, $conditionType = 'eq')
     * @param int $setId
     * @return $this
     */
    public function setAttributeSetFilter($setId)
    {
        $this->addFieldToFilter('attribute_set_id', ['eq' => $setId]);
        $this->setOrder('sort_order');
        return $this;
    }

    /**
     * Set Attribute Set Condition Type Set Filter
     *
     * @param string $setId
     * @param string $conditionType
     * @return $this
     */
    public function setAttributeSetConditionSetFilter($setId, $conditionType = 'eq')
    {
        $this->addFieldToFilter('attribute_set_id', [$conditionType => $setId]);
        $this->setOrder('sort_order');
        return $this;
    }
```
``app/code/Magento/Eav/Model/Api/SearchCriteria/CollectionProcessor/FilterProcessor/AttributeGroupAttributeSetIdFilter.php
```
public function apply(Filter $filter, AbstractDb $collection)
    {
        /** @var \Magento\Eav\Model\ResourceModel\Entity\Attribute\Group\Collection $collection */
        $collection->setAttributeSetConditionSetFilter($filter->getValue(), $filter->getConditionType());

        return true;
    }
```